src/transactions/TransactionEngineTest.java

```diff
@@ -55,5 +59,50 @@ public void testUniquenessAndDigits() {
             }
         }
     }
+    //Now I will test the createPersonMap() methods in the DataManager class by creating a csv file
+        //with a few Person objects in it then create a DataManager class and make sure that a Person map is created
+        //for Person objects that have null fields (such as address) and that a transaction ID is created for the Person
+        //objects in the csv file
+    @Test
+    public void testIDCreatedWithGetPersonMap() {
+        DataManager dataManager = new DataManager("Test-Customers.csv");
+        //now do the createPersonMap() method to create a map of Person objects with their account number
+            //as their key
+        Map<String, Person> personMap = dataManager.createPersonMap();
+        //Now I will get a few of the Person objects in the csv file and store them here to see if there are
+            //transaction IDs for them
+        Person person1 = personMap.get("abc123");
+        Person person2 = personMap.get("success789");
```

Comment:
In general, you want to write this like
```py 
# generate & test person 1 in this block
person1 = generatePerson()
assertSomething(person2)

# test stuff about person 2 over here
person2 = generateAnotherPerson()
assertSomethingElse(person2)
```
This way it’s easier to track what breaks.
Besides that, this looks good.

Reply:
Yeah this will make it easier to debug should something go wrong

Reply:
Okay, I can change that stuff tomorrow